contrib/depends/packages/openssl.mk

```diff
@@ -1,8 +1,8 @@
 package=openssl
-$(package)_version=1.0.2r
-$(package)_download_path=https://ftp.openssl.org/source/old/1.0.2
+$(package)_version=1.1.1i
+$(package)_download_path=https://www.openssl.org/source
```

Comment:
This patch looks quite different from the original https://github.com/monero-project/monero/pull/7100/commits/5b30203fe956990e9cf0fa7ad5a280b88b7683e8, and notably misses some configurations related to ARM building. This is why I prefer avoiding out-of-order cherry-picking.

Do you insist that this OpenSSL update is necessary now? If so, maybe it's better not to cherry-pick and make a minimal patch bumping the version only, so that the upstream changes can be picked up later.

Reply:
I see what you are saying now. We have no arm building support so we should not cherry pick this until we have merged that code. Then there will be easier picking in the future.  Well perhaps this is not a critical concern and can be postponed. I don't know how this library is used in the aeon code maybe it is a very insignificant dependency. hyc posted this comment about a potential flaw 1.0.2 https://github.com/monero-project/monero/pull/7100#issuecomment-744709797

Reply:
Currently Aeon uses OpenSSL only for calculating SHA256, so this vulnerability doesn't apply. But it will become relevant when we take the SSL support patches from upstream, so it may be advisable to bump it soonish.